docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -902,3 +902,29 @@ Once this is done, restart pfmon on all nodes using:
 
 NOTE: It is normal that pfmon will refuse to start on nodes B and C. Nonetheless you should execute the command on them to make sure you catch any stale process.
 
+MariaDB Galera cluster troubleshooting
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Max connections reached
++++++++++++++++++++++++
+
+In the event that one of the 3 servers reaches the maximum amount of connections (defaults to 1000), this will dead-lock the Galera cluster synchronization. In order to resolve this, you should first increase database_advanced.max_connections, then stop packetfence-mariadb on all 3 servers, and follow the steps in the section 'Recovering from a split brain' of this document using anyone of the 3 servers as your source of truth.
+
+Investigating further
+^^^^^^^^^^^^^^^^^^^^^
+
+The limit of 1000 connections is fairly high already so if you reached the maximum number of connections, this might indicate an issue with your database cluster. If this issue happens often, you should monitor the active connections and their associated queries to find out what is using up your connections.
```

Comment:
Not sure its the job of our best practices script to monitor that, especially since they run at every hour and this can explode in much less than that.

We'd need something a bit more proactive than those scripts like a dedicated monit check IMO